app/js/components/shared/Dropdown.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
```

Comment:
Same here... what do you think about moving this into a new "widgets" directory in openmrs-reactcomponents

Reply:
yeah, would do that right away!